exercises/concept/secrets/.meta/design.md

```diff
@@ -17,3 +17,17 @@ The goal of this exercise is to teach the student about bitwise operations in Ja
 This exercise's prerequisites Concepts are:
 
 - 'numbers'
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: If the student did not use the `>>>` operator in the `shiftBack` method, instruct the student to do so.
```

Comment:
Technically using the `>>` operator is also fine, right? I don't think they are _required_ to use the unsigned shift right operator, a normal shift right will work too.

Reply:
Currently, no `>>` wouldn't be fine. One of the tests will fail if they use it and the instructions lead the students to choose `>>>`.

Reply:
Ok thanks for clearing that up!